### PR TITLE
Fix change scene crash v3.6.0

### PR DESCRIPTION
**Pull request by troublemaker52025**
Re: #

### Changelog

* 

-------

### Continuous Integration

This pull request:

* [x] needs automatic test cases check.

  > Manual trigger with `@cocos-robot run test cases` afterward.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

**Comment by troublemaker52025** [comment]
@cocos-robot run test cases

File: native/cocos/core/assets/SimpleTexture.cpp [hunk 1]
```diff
@@ -181,7 +181,6 @@ gfx::Texture *SimpleTexture::createTextureView(gfx::Device *device) {
 
 void SimpleTexture::tryDestroyTexture() {
     if (_gfxTexture != nullptr) {
-        _gfxTexture->destroy();
```
Comment (star-e) [inline native/cocos/core/assets/SimpleTexture.cpp]: I think we should destroy _gfxTexture.

Comment (star-e) [reply native/cocos/core/assets/SimpleTexture.cpp]: The one who called tryDestroyTexture is responsible for the double free.

File: native/cocos/core/assets/SimpleTexture.cpp [hunk 2]
```diff
@@ -190,7 +189,6 @@ void SimpleTexture::tryDestroyTexture() {
 
 void SimpleTexture::tryDestroyTextureView() {
     if (_gfxTextureView != nullptr) {
-        _gfxTextureView->destroy();
```
Comment (jk20012001) [inline native/cocos/core/assets/SimpleTexture.cpp]: _gfxTextureView also needs to be destroyed?

**Pull request closed without merging**